draft-ietf-quic-recovery.md

```diff
@@ -1142,6 +1142,21 @@ Pseudocode for OnPacketSent follows:
      SetLossDetectionTimer()
 ~~~
 
+## On Receiving a Packet
+
+When a packet is received, it may allow the server to send
+if the server was previously amplification limited.
+
+Psuedocode for OnPacketReceived follows:
+
+~~~
+OnPacketReceived(packet):
+  // If this packet unblocks the server, arm the
```

Comment:
This is fine.  The alternative I considered was:

```
  if (server and peer address is not validated):
    was_blocked = aa_bytes_sent >= aa_bytes_received * 3
    aa_bytes_received += datagram.size()
    if was_blocked:
        SetLossDetectionTimer()
```
But this is fine.  In order for that to work, you need to have global state variables for aa_bytes_sent and aa_bytes_received, which you don't otherwise have.

Reply:
Yeah, I was hoping to avoid that because I think it'd be a substantial amount of extra pseudocode.